core/metacling/src/CMakeLists.txt

```diff
@@ -128,6 +128,12 @@ if(MSVC)
       __std_terminate
       cling_runtime_internal_throwIfInvalidPointer
   )
+  if(CMAKE_CXX_STANDARD GREATER 14)
```

Comment:
Isn't that always the case in master?

Reply:
> Isn't that always the case in master?

Nope, the minimal is 14, and the test is for >14

Reply:
😫